### PR TITLE
Allow `clean-pinned-issues` to be disabled

### LINKED ISSUE #3773: Add option to disable the pinned issue list layout
https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc

image

image

image

When searching for `pin` I cannot find any option to disable the pinned issue list layout feature.

I personally prefer the default one from GitHub but want to use all other features of `refined-github`

### PR DESCRIPTION
**Pull request by FbFDestro**
1. LINKED ISSUES:
   Closes #3773

### CODE REVIEW

File: source/features/clean-pinned-issues.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const pinned_issues_container = select('.js-pinned-issues-reorder-container');
+	if (pinned_issues_container) {
+		pinned_issues_container.classList.add('clean-pinned-issue');
+	}
```
Comment (fregante) [inline source/features/clean-pinned-issues.tsx:11]: You can add the class to `body`, unconditionally

File: source/features/clean-pinned-issues.css [hunk 2]
```diff
@@ -1,10 +1,10 @@
 /* Changes the layout of pinned issues from side-by-side to a standard list. */
-.js-pinned-issues-reorder-container .f4 {
+.clean-pinned-issue .js-pinned-issues-reorder-container .f4 {
 	display: none !important; /* Hide title */
 }
 
 @media (min-width: 700px) {
-	:root .js-pinned-issues-reorder-list {
+	:root .clean-pinned-issue .js-pinned-issues-reorder-list {
```
Comment (fregante) [inline source/features/clean-pinned-issues.css:7]: `:root` can likely be dropped now in both places. Can you make sure that it works?

**Review by fregante — APPROVED** [review]

**Review by yakov116 — APPROVED** [review]

**Comment by yakov116** [comment]
@FbFDestro can you fix the conflict? Also don't open PR's from the master branch

**Comment by FbFDestro** [comment]
@yakov116 the conflict was fixed! :)

Thank you for the advice of not opening PR's from the master branch, now I see that every new commit to the master on my fork would come to this PR and that is not desirable, next time I will create new branches :D

**Comment by yakov116** [comment]
@FbFDestro Thanks for contributing!

**Pull request merged**